### DISCUSSION TITLE
Aggiunta voti

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#645] [Richieste di aiuto]
Salve sarebbe possibile aggiungere un voto diverso da quelli presenti ?
Grazie.

**Comment by trinko** [comment]
Ogni modifica è possibile. Poi, a seconda di quello che intendi fare, può essere sufficiente modificare solo il template, oppure può essere necessario modificare anche dei file PHP, o altro ancora.

Vedi ad esempio [questa discussione](https://github.com/iisgiua/giuaschool/discussions/641)

**Reply by Davide031** [reply]
ok, se voglio creare il voto-  (con valore 4) che esce il simbolo - e il colore rosso e il voto + (con valore 6) con il simbolo + e la casella verde come posso fare?
